config/esp32/components/chip/Kconfig

```diff
@@ -516,6 +516,51 @@ menu "CHIP Device Layer"
 
     endmenu
 
+    menu "Ethernet Options"
+        visible if ENABLE_ETHERNET_TELEMETRY
+
```

Comment:
help here depending on what type of ethernet is supported

Reply:
I added help under ENABLE_ETHERNET_TELEMETRY option